examples/plotter.py

```diff
@@ -0,0 +1,106 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""These functions compare properties of different molecules."""
+import numpy
+import pylab

```

Comment:
There is no code that generates a warning.

If you add a new line for each doc string in the rest of FermiLib (which all follows a different convention) then I will match the "new" convention here. Otherwise, I am going to be consistent with the rest of the library.